.java-version

```diff
@@ -0,0 +1 @@
+21.0
```

Comment:
A quoi il sert ce fichier ?

Attention si on dit qu'on supporte Java 17 il faut utiliser le JDK 17 pour la pipeline et pas le 21 car sinon on peut se retrouver à utiliser des API du JDK 21 qui ne sont pas dans le JDK 17

Reply:
C'est pour `jenv` ⬇️ 

> Command-line tool for managing Java environments on your machine. It allows you to switch between different Java versions seamlessly.
When `jenv` is configured in a project directory, it looks for the .java-version file to determine the Java version to use for that project.

OK pour la pipeline je modifie ça